modules/twinkleprotect.js

```diff
@@ -132,21 +157,42 @@ Twinkle.protect.fetchProtectionLevel = function twinkleprotectFetchProtectionLev
 					expiry: protection.expiry,
 					cascade: protection.cascade === ''
 				};
+				// logs report last admin who made changes to either edit/move/create protection, regardless if they only modified one of them
+				if (!adminEditDeferred) {
+					adminEditDeferred = Twinkle.protect.fetchProtectingAdmin(api, mw.config.get('wgPageName'));
+				}
 			}
 		});
 
+		Twinkle.protect.hasStableLog = !!stableData[0].query.logevents.length;
+
 		if (page.flagged) {
+			// note that stable settings aren't logged when page is moved, so we don't need to use fetchProtectingAdmin
```

Comment:
This seems to longer be the case, being taken care of in [T59912](https://phabricator.wikimedia.org/T59912) a year after your original pull, so we'll have to redo this bit.

Reply:
I think it should still be doable with `stableData`, is that right?  If we check through `stableData[0].query.logevents` for the first entry without `"action": "move_stable"`.  Annoyingly, `leaction` seems to only allow one parameter (`stable/config` or `stable/modify` but not both) and doesn't allow negative filtering (can't exclude `stable/move_stable` or `protect/move_prot`) which would make this whole thing easier.

Reply:
```suggestion
```